lib/options/options-lib_dcu.robot: fix

wiktormowinski opened the pull request:
working on a fix to this https://github.com/Dasharo/open-source-firmware-validation/issues/772

wiktormowinski committed:
lib/options/options-lib_dcu.robot: fix

Signed-off-by: Wiktor Mowinski <wiktor.mowinski@3mdeb.com>

wiktormowinski committed:
lib/options/options-lib_dcu.robot: actual fix

Signed-off-by: Wiktor Mowinski <wiktor.mowinski@3mdeb.com>

mkopec commented:
Can you squash the commits and add a proper commit message? As-is, I don't know what issue this is supposed to fix based only on the commits

wiktormowinski commented:
@mkopec of course, updated pr description and squashed commits.
the issue in question was: https://github.com/Dasharo/open-source-firmware-validation/issues/772
